### PR TITLE
Move less install to correct spot

### PR DESCRIPTION
**Pull request by acbramley**

### CODE REVIEW

File: cli/Dockerfile [hunk 1]
```diff
@@ -28,7 +28,8 @@ RUN apk --update --no-cache add \
       openssh-client \
       patch \
       git \
-      jq
+      jq \
+      less
```
Comment (kimpepper) [inline cli/Dockerfile:32]: can we have this in alphabetically order please :pray:

Comment (nickschuch) [reply cli/Dockerfile:32]: no

Comment (acbramley) [reply cli/Dockerfile:32]: I was going to and then saw the rest weren't so 🤷

File: cli/tests.yml [hunk 2]
```diff
@@ -20,3 +20,9 @@ commandTests:
     command: "redis-cli"
     args: ["--version"]
     exitCode: 0
+
+  - name: 'correct less binary installed'
+    command: "less"
+    args: ["--version"]
+    expectedOutput: ["Mark Nudelman"]
```
Comment (acbramley) [inline cli/tests.yml:27]: Shoutouts Mark

**Pull request merged**